### PR TITLE
[DT-930] - Include namehash to API responses

### PR DESCRIPTION
**Pull request by tunguyennnnn**
## Background

Currently the following endpoints don't return the tokenId (eip37 hash of the domain):
`/domains`, `/domains/:name`, `/metadata/:domainOrToken`, `/reverse/:address`, and `/reverse/query`

[Linear Issue](https://linear.app/unstoppable-domains/issue/DT-930/resolution-service-api-add-tokenid-to-responses)

## Changes

* added the tokenId to the response of the APIs
* added `test:watch` script for development

For example:
- [ ] Rebase against master once #9999 is merged
- [ ] Fix flakey test
-->

## Code Review

This Pull Request was thoroughly reviewed and meets all Code Review Standards pertaining to:

- [ ] **Implementation** - satisfied acceptance criteria
- [ ] **Communication** - notified engineers/stakeholders about impactful changes
- [ ] **Error handling** - handled, logged & alerted on errors
- [ ] **Documentation** - wrote readable code & commits, documented unintuitive code
- [ ] **Testing** - unit & E2E test coverage, tested in staging, DB migrations backwards compatible
- [ ] **Security** - sanitized inputs, vetted dependencies, validated/secured API requests, no committed secrets
- [ ] **Performance** - optimized expensive algorithms & database queries

Please select all **applied** standards relevant to this PR.
## Confirmation

- [ ] **Assignee Confirmation** - I (the author) have filled out the template above
- [ ] **Reviewer Confirmation** - I (a/the reviewer) confirm 1) the author has filled out the template and checked the box that says **Assignee Confirmation** 2) I reviewed the code and selected all applicable items in the code review section.

### CODE REVIEW

File: src/controllers/DomainsController.ts [hunk 1]
```diff
@@ -223,6 +203,7 @@ export class DomainsController {
           records: resolution.resolution,
           meta: {
             domain: domain.name,
+            tokenId: domain.node,
```
Comment (DancingAxolotl) [inline src/controllers/DomainsController.ts:206]: I'd suggest renaming this to `namehash` or converting the hex value of `domain.node` to decimal.

It's a bit confusing, but technically:
 - `tokenId` is the ERC721 token id which is a decimal number (uint256)
 - `namehash` is the EIP-137 namehash which is a hex number

In our case it's the same value, just represented differently so this is okay. But for consistency with other naming services we should either return the token id in the correct format or rename this field to `namehash`.

And I don't really know why it was named `node` in the domain entity. It's probably a historical name left over from the website entity.

Comment (tunguyennnnn) [reply src/controllers/DomainsController.ts]: FV, discussion is in [slack](https://unstoppabledomains.slack.com/archives/C022W48Q10C/p1670514009283099)

File: src/services/Resolution.ts [hunk 2]
```diff
@@ -1,9 +1,15 @@
+import { BigNumber } from 'ethers';
+import { In } from 'typeorm';
```
[the remark this answers is not in the document]
Comment (tunguyennnnn) [reply src/services/Resolution.ts:2]: yes it's used in `getReverseResolution`, I just moved the import to group the lib import and our files import separately for code readability

**Review by enaqx — APPROVED** [review]
LGTM!

**Pull request merged**